### PR TITLE
Jr-payment type

### PR DESCRIPTION
**Pull request by JordanRosas**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
I was assigned the PaymentType issue ticket: User should be able to GET a list, and GET a single item.
All methods in the controller are passing successfully
*Added sql parameters to the post method in the payment type controller
GET
GET/ID
POST
PUT
DELETE

Tests are all passing successfully!

Fixes # (issue)
#3 
## Type of change
Testing, Controller, Model
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ X ] This change requires a documentation update

# Testing Instructions
1. `git fetch --all`
2. `git checkout jr-paymentType`
3. Open the testing project `PaymentTypeTest.cs`
4. Open the test explorer 
5. `Run All` and ensure that all tests are passing successfully
6. Open Postman
7.  perform a GET, PUT, POST, and DELETE in postman to make sure all methods are working properly

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests

# Checklist:

- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] I have commented on my code, particularly in hard-to-understand areas
- [ X ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ X ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by bramosja — APPROVED** [review]
Looks good!

**Review by allisonkatecollins — APPROVED** [review]
looks gud

File: BangazonAPI/Controllers/PaymentTypeController.cs [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                    }
+                    reader.Close();
+                    return Ok(specific_PaymentType);
+                }
+            }
+        }
+
+        // POST: api/PaymentType
+        [HttpPost]
+        public async Task<IActionResult> Post([FromBody] PaymentType paymentType)
```
Comment (askingalot) [inline BangazonAPI/Controllers/PaymentTypeController.cs:97]: What happens if `paymentType.Name` is null?

File: BangazonAPI/Controllers/PaymentTypeController.cs [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO PaymentType ([Name], AcctNumber, CustomerId)
+                                        OUTPUT INSERTED.id
+                                        Values ('@name', @AcctNumber, @CustomerId)";
```
Comment (askingalot) [inline BangazonAPI/Controllers/PaymentTypeController.cs]: why is `'@name'` quoted?

**Review by askingalot — CHANGES_REQUESTED** [review]

**Review by askingalot — APPROVED** [review]

**Pull request merged**